tests/certification/pubsub/pulsar/pulsar_test.go

```diff
@@ -1243,6 +1256,15 @@ func (p *pulsarSuite) client(t *testing.T) (pulsar.Client, error) {
 	opts := pulsar.ClientOptions{
 		URL: "pulsar://localhost:6650",
 	}
+
+	if p.tlsEnabled {
+		opts = pulsar.ClientOptions{
+			URL:                        "pulsar+ssl://localhost:6651",
+			TLSAllowInsecureConnection: true,
```

Comment:
Isn't this the opposite of what you want?

Reply:
No, this PR add the ability to connect to pulsar over an SSL connection with the TLSAllowInsecureConnection (self signed cert)